upgrading/rosa-hcp-upgrading.adoc

```diff
@@ -6,20 +6,13 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-[id="rosa-hcp-lifecycle-policy_{context}"]
-== Life cycle policies and planning
+You can upgrade {hcp-title-first} clusters by using individual upgrades through the {product-title} (ROSA) command line interface (CLI), `rosa`. You can also specify if you want to upgrade just the control plane, a single machine pool, or all machine pools and the control plane.
```

Comment:
```suggestion
Upgrade {hcp-title-first} clusters by individually upgrading the hosted control plane and the machine pools with the ROSA command line interface (CLI), `rosa`. 

Use one of the following methods to upgrade your HCP clusters:
* Upgrade only your hosted control plane. This does not impact your worker nodes. 
* Upgrade only your machine pool. This initiates a rolling reboot of a specific machine pool and temporarily impacts the worker nodes on the specific machine pool. It will not impact all your worker nodes if you have multiple machine pools.
* Upgrade your hosted control plane first and then your machine pool.
[NOTE]
====
If you want to upgrade both your hosted control plane and your machine pool, you must upgrade the the hosted control plane first.
====
```
I'm not sure how you want to format this, but I *think* this is mostly accurate based on what the SRE gave you.